jenkins/ci.opensuse.org/openstack-devstack.yaml

```diff
@@ -39,7 +48,7 @@
               virsh destroy cleanvm
               find /mnt/cleanvmbackup -mtime +5 -type f | xargs --no-run-if-empty rm
               # backup /dev/vg0/cleanvm disk image
-              file=/mnt/cleanvmbackup/${BUILD_NUMBER}-devstack.raw.gz
+              file=/mnt/cleanvmbackup/$BUILD_NUMBER-devstack.raw.gz
```

Comment:
Why that? What was wrong with the ${} notation?

Reply:
Ok, got the answer meanwhile: it is now part of a template section and the {} are interpreted differently.

Reply:
yeah, the templates are preprocessed so it tries to expend the variables within the curly braces.
It was a long time since I worked with JJB, I remember to use double {} curly braces to escape them, but I was checking how openstack jobs were using it and seems that they prefer to add the variables in the format `$variable` https://github.com/openstack-infra/project-config/blob/before-jenkins-config-removed/jenkins/jobs/ansible-role-jobs.yaml